python_standards.md

```diff
@@ -0,0 +1,167 @@
+- Use CAPITALS to define constants or globals that will be used in your entire program or module
+- Use lowercase_with_underscores for local variables, member attributes or methods, function arguments and parameters
+- Use PascalCase for classes
+- Use short lowercase names for scripts
+
+#### Global variables and namespaces
+In Python, variable lookups go up from local namespace to parent namespace to global namespace. Below, python looks in the local namespace of multiply and cannot find GLOBALVAR so it looks in the parent scope
+```{python}
+GLOBALVAR = 12345  
+
+def multiply(x):  
+    return x * GLOBALVAR # GLOBALVAR = 12345
+```
+If you declare a variable inside a local scope, then it is immediately local unless you use the global keyword (which works at the module level)
+```{python}
+GLOBALVAR = 12345  
+
+def multiply(x):  
+    GLOBALVAR = 1  
+    return x * GLOBALVAR # GLOBALVAR = 1  
+
+def changeGlobal(x):  
+    global GLOBALVAR        # GLOBALVAR in this scope references the global GLOBALVAR  
+    GLOBALVAR = 5678        # changes GLOBALVAR  
+    return x * GLOBALVAR    # GLOBALVAR = 5678  
+
+print(GLOBALVAR)            # prints 12345  
+x = multiply(2)             # x = 24690  
+print(GLOBALVAR)            # prints 12345  
+y = changeGlobal(2)         # y = 11356  
+print(GLOBALVAR)            # prints 5678
+```
+### Documentation
+#### Docstrings
+For more info on docstring best practices, see [PEP 257](https://peps.python.org/pep-0257/).
+
+A docstring is a string literal that occurs as the first statement in a module, function, class, or method definition. Such a docstring becomes the ```__doc__``` special attribute of that object.
+
+All modules should normally have docstrings, and all functions and classes exported by a module should also have docstrings. Public methods (including the ```__init__``` constructor) should also have docstrings. A package may be documented in the module docstring of the ```__init__.py``` file in the package directory.
+
+For consistency, always use ```"""triple double quotes"""``` around docstrings. Use ```r"""raw triple double quotes"""``` if you use any backslashes in your docstrings. 
+
+#### Comments
+Block comments generally apply to some (or all) code that follows them, and are indented to the same level as that code. Each line of a block comment starts with a # and a single space (unless it is indented text inside the comment).
+
+Paragraphs inside a block comment are separated by a line containing a single #.
+
+Use inline comments sparingly. An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.
+
+Inline comments are unnecessary and in fact distracting if they state the obvious. Don’t do this:
+```{python}
+x = x + 1         # Increment x
+```
+But sometimes, this is useful:
+```{python}
+x = x + 1         # Compensate for border
+```
+## Packages
```

Comment:
Any others for the categories Reggie included in the [R standards](https://github.com/HHS/acf-datasurge-standards/blob/main/r_standards.md)?